PowerShellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingServiceUtilities.cs

```diff
@@ -255,6 +235,32 @@ private static void ProvideDteVariable(Runspace runspace)
             }
         }
 
+        private static void ProvideProfileVariable()

```

Comment:
I do not immediately see a reason why it could not be static, but I was following the convention of the earlier ProvideDteVariable variable which is static.